manual/examples/lots-of-objects-animated.html

```diff
@@ -58,7 +58,7 @@
 import * as THREE from 'three';
 import * as BufferGeometryUtils from 'three/addons/utils/BufferGeometryUtils.js';
 import {OrbitControls} from 'three/addons/controls/OrbitControls.js';
-import {TWEEN} from 'three/addons/libs/tween.module.min.js';
+import TWEEN from 'three/addons/libs/tween.module.js';
```

Comment:
There are still pages in the manual than require and update. Please search for them in the root folder with your editor. There should be six pages in total.

Reply:
Thanks, I had searched for `tween.module` since there are many references to the word "tween," but it looked like the manuals were outdated and were importing from `tween.min.js`. Should be fixed now, let me know if I missed others.